indexer/altitude_loader.cpp

```diff
@@ -54,7 +54,10 @@ AltitudeLoader::AltitudeLoader(MwmValue const & mwmValue)
   }
 }
 
-bool AltitudeLoader::HasAltitudes() const { return m_header.m_minAltitude != kInvalidAltitude; }
+bool AltitudeLoader::HasAltitudes() const
+{
+  return m_reader != nullptr && m_header.m_minAltitude != kInvalidAltitude;
```

Comment:
How about to make this method public and test it too? Or, even better, declare test code as a friend to the class and test it?